common/index.ftd

```diff
@@ -73,3 +77,39 @@ $processor$: pr.toc
   - Footer Link 01: /
   - Footer Link 01: /
   - Footer Link 01: /
+
+
+
+
+
+
+
+
+
+
+-- record author-meta:
+optional caption title:
+optional string profile:
+optional string company:
+optional string bio-url:
+optional ftd.image-src image:
+optional body body:
+
+
+
+
+
+-- record post-meta:
+optional caption title:
+optional string published-on:
+optional ftd.image-src post-image:
+optional body body:
+optional string post-url:
+optional author-meta author:
+optional string info: 
+optional string cta-txt1: 
+optional string cta-url1:
+optional string cta-txt2: 
+optional string cta-url2:
```

Comment:
```suggestion
optional string cta-primary: 
optional string cta-primary-url:
optional string cta-secondary: 
optional string cta-secondary-url:
```

Reply:
@MeenuKumari28 @priyanka9634 plz add these properties inside featured-post. And remove it from here.